modules/network/subnets.tf

```diff
@@ -109,18 +120,16 @@ resource "oci_core_subnet" "oke" {
 
   lifecycle {
     # TODO reflect default security_list_id instead of ignore
-    ignore_changes = [security_list_ids, freeform_tags, defined_tags, dns_label]
+    ignore_changes = [security_list_ids, freeform_tags, defined_tags, dns_label, display_name, cidr_block]
   }
 }
 
 # Create an associated security list for subnets when enabled
 # e.g. for load balancers to prevent CCM management of default security list
 resource "oci_core_security_list" "oke" {

```

Comment:
So, this is to avoid the security rules going into the default security list?

Reply:
Can you also add the default security list in troubleshooting mode? Sometimes, when we are experimenting, we just want to know if things will work first before going hunting for the rules.